client/reader/sidebar/expandable.jsx

```diff
@@ -0,0 +1,125 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import classNames from 'classnames';
+
+/**
+ * Internal Dependencies
+ */
+import Gridicon from 'components/gridicon';
+import Button from 'components/button';
+import Count from 'components/count';
+import SidebarMenu from 'layout/sidebar/menu';
+import SidebarHeading from 'layout/sidebar/heading';
+
+const ExpandableSidebarMenu = React.createClass( {
+
+	propTypes: {
+		title: React.PropTypes.oneOfType( [ React.PropTypes.string, React.PropTypes.element ] ).isRequired,
+		count: React.PropTypes.number,
+		addPlaceholder: React.PropTypes.string,
+		onAddSubmit: React.PropTypes.func
+	},
+
+	getInitialState() {
+		return {
+			isAdding: false
+		};
+	},
+
+	getDefaultProps() {
+		return {
+			expanded: false
+		};
+	},
+
+	onClick() {
+		if ( this.props.disabled ) {
+			return;
+		}
+
+		if ( this.props.onClick ) {
+			this.props.onClick();
+		}
+	},
+
+	renderContent() {
+		return (
+			<ul className="sidebar__menu-list">
+				{ this.props.children }
+			</ul>
+		);
+	},
+
+	renderHeader() {
+		const headerClasses = classNames( 'sidebar__menu-header' );
+		return (
+			<div className={ headerClasses } onClick={ this.onClick }>
```

Comment:
Can we avoid the wrapper div and pass classNames and an onClick to SidebarHeading?